pkg/shell/hosts.jsx

```diff
@@ -307,7 +306,6 @@ export class CockpitHosts extends React.Component {
                                 item_render={render}
                                 sorting={(a, b) => true}
                                 filtering={this.filterHosts}
-                                current={label}
```

Comment:
So it seems CockpitNav does not use `current` at all, can we then split this removal off?

Reply:
No, that was a mistake unfortunately. It uses `current` to notice when navigation has happened, and then resets the search field.

This might be a good example of hard to understand magic that we want to get rid of. There might be a signal on the ShellState that announces navigation, or we might have a `navigation_counter`, or something more obvious.